flow/migration/__init__.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2021 The Regents of the University of Michigan
```

Comment:
I honestly think it is a terrible idea to duplicate all of this logic here. Is it really that hard to adjust the existing framework to perform the migration?